include/vcpkg/base/messages.h

```diff
@@ -751,6 +752,10 @@ namespace vcpkg
     DECLARE_MESSAGE(ExpectedPortName, (), "", "expected a port name here");
     DECLARE_MESSAGE(ExpectedTripletName, (), "", "expected a triplet name here");
     DECLARE_MESSAGE(ExtendedDocumentationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
+    DECLARE_MESSAGE(FailedToFindDependencyAbi,
```

Comment:
Hmmm not sure how I feel about this one; I think it should be a `Debug::` message.

What's going on here is, when computing the ABI hash for a package, that contains the ABI hashes of the dependencies:

![image](https://user-images.githubusercontent.com/1544943/179327128-231c7bce-dbfb-418e-b198-e605bff4b51d.png)

To get this error message, vcpkg would have to somehow decide that this dependency needed to be installed, then also somehow be unable to find the dependency when calculating this hash. For that to happen something has seriously gone off the rails and I don't think a message beyond one targeted at vcpkg's developers themselves is going to do much...